index.htm

```diff
@@ -342,52 +342,15 @@ <h3>How can I participate?</h3>
                 <h4 class="text-bold">
                     Download the latest Gridcoin Research client
                 </h4>
-                <p>
-                	Current Wallet Version: 4.0.3.0 (Leisure)
+                <p id="wallet-version">
                 </p>
             </div>
-        </div>
-        <div class="row">
             <!--GRC Research Downloads-->
-            <div class="col-xs-12 col-sm-6">
-              <h4>Windows 64-bit</h4>
-              <a class="btn btn-round grcbtn" href="https://github.com/gridcoin-community/Gridcoin-Research/releases/download/4.0.3.0/gridcoin-4.0.3-win64-setup.exe">
-                  Download 64-bit Installer
-              </a><br />
-              <a class="btn btn-round grcbtn" href="https://github.com/gridcoin-community/Gridcoin-Research/releases/download/4.0.3.0/gridcoin-4.0.3-win64-setup.exe.sha256">
-                  64-bit SHA256 Checksum
-              </a>
-            </div>
-            <div class="col-xs-12 col-sm-6">
-              <h4>Windows 32-bit</h4>
-              <a class="btn btn-round grcbtn" href="https://github.com/gridcoin-community/Gridcoin-Research/releases/download/4.0.3.0/gridcoin-4.0.3-win32-setup.exe">
-                  Download 32-bit Installer
-              </a><br />
-              <a class="btn btn-round grcbtn" href="https://github.com/gridcoin-community/Gridcoin-Research/releases/download/4.0.3.0/gridcoin-4.0.3-win32-setup.exe.sha256">
-                  32-bit SHA256 Checksum
+            <div class="col align-self-center">
+              <a class="btn btn-round grcbtn" href="https://github.com/gridcoin-community/Gridcoin-Research/releases/latest">
+                  Download via Github.com!

```

Comment:
Do we need an exclamation point?  😛

Reply:
Nope. Probably not. Do we need the .com? I've been wondering myself whether it should be 'Download via Github' or 'Download from Github' or github.com etc etc.

I'm leaning towards 'Download via Github' myself

Reply:
I think "Download via Github" is best as well